calendar/page.js

```diff
@@ -82,31 +88,41 @@ class CalendarHandler {
       this.calendar.updateEvent(record.id, CALENDAR_NAME, {backgroundColor: CalendarHandler._selectedColor});
       this._selectedRecordId = record.id;
       this.calendar.setDate(record.startDate);
-      const dom = document.querySelector('.toastui-calendar-time');
-      const middleHour = record.startDate.getHours()
-        + (record.endDate.getHours() - record.startDate.getHours()) / 2;
-      dom.scrollTo({top: (dom.clientHeight / 24) * middleHour, behavior: 'smooth'});
+      updateUIAfterNavigation();
+      if (this.calendar.getViewName() !== 'month') {
+        //Scroll to the middle of the event if it's not month view.
+        //In some cases, an event is not visible even if a valid day is focused - for example, when event is in the
+        //last hour of the day, so to make it visible, we need to scroll to the middle of the event.
+        const dom = document.querySelector('.toastui-calendar-time');
+        const middleHour = record.startDate.getHours()
+          + (record.endDate.getHours() - record.startDate.getHours()) / 2;
+        dom.scrollTo({top: (dom.clientHeight / 24) * middleHour, behavior: 'smooth'});
+      }
     }
   }
 
   // change calendar perspective between week, month and day.
   changeView(calendarViewPerspective) {
     this.calendar.changeView(calendarViewPerspective);
+    document.getElementById('calendar-title').innerText = getMonthName();
```

Comment:
Could this be a call to `updateUIAfterNavigation`

Reply:
In current moment there is no point - changing perspective should persist day that calendar is focused on, so it should not change neither month nor year that is showed in header. But in future other UI element can be updated after navigation, so I'll add it

Reply:
I'm saying that if you think of `updateUIAfterNavigation` as being in charge of updating the UI after some significant change, there's no need to duplicate the `document.getElementById('calendar-title').innerText = getMonthName();` code here - that can be the responsibility of `updateUIAfterNavigation`.